audioipc/src/handle_passing.rs

```diff
@@ -175,16 +175,17 @@ where
             }
         }
 
-        let mut got_handles = false;
-        if let Some((handles, target_pid)) = item.platform_handles() {
-            got_handles = true;
+        // Take handle ownership here.
+        let handles = item.platform_handles();
+        if let Some((handles, target_pid)) = handles {
+            // TODO: This could leak target handles if a duplicate fails - make this more robust.
```

Comment:
This issue exists in the current code also.  Fixing https://github.com/mozilla/audioipc-2/issues/68 will address this.